filter_plugins/oo_filters.py

```diff
@@ -881,14 +881,13 @@ def oo_image_tag_to_rpm_version(version, include_dash=False):
         """
         if not isinstance(version, basestring):
             raise errors.AnsibleFilterError("|failed expects a string or unicode")
-        # TODO: Do we need to make this actually convert v1.2.0-rc1 into 1.2.0-0.rc1
-        # We'd need to be really strict about how we build the RPM Version+Release
         if version.startswith("v"):
-            version = version.replace("v", "")
+            version = version[1:]
+            # Strip release from requested version, we no longer support this.
             version = version.split('-')[0]
```

Comment:
Added a task for this, will need more details on these tags you're referring to.